main.py

```diff
@@ -90,7 +90,7 @@ def get_variables(context) -> Variables:
     )
     var.function_schedule = properties.get("function_schedule", "*/15 * * * *")
     var.function_available_memory_mb = int(
-        properties.get("function_available_memory_mb", "256")
+        properties.get("function_available_memory_mb", "512")
```

Comment:
nit: make it a constant, or even better, get it from env var

Reply:
what do you mean by constant/env var

btw, This code is analogous to https://github.com/observeinc/terraform-google-collection/blob/main/variables.tf